src/store/modules/user/actions.ts

```diff
@@ -29,12 +29,12 @@ const actions: ActionTree<UserState, RootState> = {
           return Promise.reject(new Error(resp.data._ERROR_MESSAGE_));
         }
       } else {
-        showToast(translate('Something went wrong'));
+        showToast(translate('Something went wrong'), getResponseError(resp));
         console.error("error", resp.data._ERROR_MESSAGE_);
         return Promise.reject(new Error(resp.data._ERROR_MESSAGE_));
       }
     } catch (err) {
-      showToast(translate('Something went wrong'));
```

Comment:
I think this is not needed. We should not share any server errors to unauthenticated user